### PR TITLE
FIRST CAPSTONE PROJECT

### PR DESCRIPTION
**Pull request by aubinleyoung**
### FIRST CAPSTONE PROJECT

- [ ]  This capstone project is based on an online website for Events.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+                <li><a href="">News</a></li>
+            </ul>
+
+        </div>
+    </header>
+    <section class="head-line-section">
+        <div class="head-line-container">
+            <h2 class="orange">"Hello sharing world"</h2>
+            <h1 class="big-title">GOLDEN EVENT WEBSITE </h1>
+
```
Comment (HAMMAS-SALEEM) [inline index.html:65]: ### Your Title
big-title

### Design Title
big-title

- [ ] Your title is not using the correct form which is COCOGOOSE. Kindly use COCOGOOSE font for this title to follow the design 👍

File: index.html [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+            <p class=" headline-p">Lorem ipsum dolor sit, amet consectetur adipisicing elit. Aliquam atque, similique
+                nobis quasi facilis dignissimos inventore delectus, eaque pariatur modi est laudantium doloribus laborum
+                quis ipsa non sint facere explicabo?</p>
```
Comment (HAMMAS-SALEEM) [inline index.html:68]: ### Your Paragraph
box-background

### Design Paragraph
box-background

- [ ] Your paragraph box should have a background color and white border which you can see in the design 🤔 Kindly follow the Cindy's design 😊

File: index.html [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+            <h2 class="bold">2023.01.01 Happy new year</h2>
+            <p class="">Lorem ipsum dolor sit amet consectetur adipisicing elit. Unde, aut.</p>
```
Comment (HAMMAS-SALEEM) [inline index.html]: ### Your Project
day-time

### Design Project
event-day-time

- [ ] You can compare both pictures and you will find out that you are not following the correct **color** and **font**. Kindly use the correct color, font and font-size 👍 to follow the CIndy design 😊

File: about.html [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+          <li><a href="">Support</a></li>
+          <li><a href="">News</a></li>
+      </ul>
+      </div>
+    </header>
+    <!-- about-section-one -->
+    <section class="about-section">
+      <div class="about-us-container">
+        <h2 class="hello-text">"Hello sharing world"</h2>
+        <h1 class="orange about-title">GOLDEN EVENT WEBSITE</h1>
```
Comment (HAMMAS-SALEEM) [inline about.html:80]: ### Your Wave
short-title-about-page

### Design Wave
short-title-about-page

- [ ] You can see the you are not using the correct color. Kindly use the correct follow the design 😊

Comment (HAMMAS-SALEEM) [inline about.html:80]: ### Your Title
big-title-about-page

### Design Title
big-title-about-page

- [ ] Your title is not using the correct form which is COCOGOOSE. Kindly use COCOGOOSE font for this title to follow the design 👍

File: index.html [hunk 5]
```diff
@@ -0,0 +1,172 @@
...
+                    <li><a href="">News</a></li>
+                    <li class="ge-campaigne"><a href="">GE campaign</a></li>
+
+                </ul>
+            </nav>
+
+        </div>
+        <button class="menu-button">
+            <img src="assets/icones/menu.png" alt="menu" width="45" height="45">
+        </button>
```
Comment (HAMMAS-SALEEM) [inline index.html:44]: ### Your Burger Icon
nav

### Design Burger Icon
nav

- [ ] You can see that in the design burger icon has a background but you don't have it. Kindly follow the design and add a background to this section 😊

File: main.js [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+      <div class="member-info">
+          <h3>${table[3].name}</h3>
+          <em class="orange italic">${table[3].function}</em>
+          <p>${table[3].description}</p>
+      </div>
+  </div>
+  </div>
+  `;
+  return memberBlock;
+});
```
Comment (HAMMAS-SALEEM) [inline main.js]: - [ ] You can see that you are repeating the code again and again. You could have done something similar which is shown below in order to not the code ⬇️

```javascript
const memberBlock = document.querySelector('.member-desktop');
table.forEach((m) => {
  memberBlock.innerHTML += `
     <div class="chess-bg-container">
     <img src="assets/icones/chess.jpg" alt="chess-bg" class="chess-bg" >
     <div class="member">
     <div class="member-image">
      
     <img src="${m.image}" alt="aubin" >
       </div>
       <div class="member-info">
           <h3>${m.name}</h3>
           <em class="orange italic">${m.function}</em>
           <p>${m.description}</p>
       </div>
   </div>
     </div>`;
  return memberBlock;
});
```

File: README.md [hunk 7]
```diff
@@ -1 +1,131 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
+
```
Comment (HAMMAS-SALEEM) [inline README.md]: - [ ] You link is broken. Kindly fix this link to make your readme file descriptive and professional 😊

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

## Hey @aubinleyoung :wave:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following Github flow
✔️ Following some requirements

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by anagudelogu — APPROVED** [review]
# STATUS: APPROVED! 🍾 🎆 🥇 

Hi @aubinleyoung, 
I'm Andres.

You did a great job working on this milestone, your project is complete! 🚀 
There is nothing else to say other than... it's time to merge it 🤖  
Congratulations! 🎉 



## Highlights
✅ Great use of GitHub flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Great UI design. 💻 
✅ Descriptive PR title, summary and commit messages. 💬 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**